test/hotspot/jtreg/vmTestbase/nsk/share/ClassUnloader.java

```diff
@@ -244,32 +250,15 @@ public void loadClass(String className, String classDir) throws ClassNotFoundExc
      */
     public boolean unloadClass(ExecutionController stresser) {
 
-        is_reclaimed = false;
-
         // free references to class and class loader to be able for collecting by GC
-        long waitTimeout = (customClassLoader == null) ? 0 : WAIT_TIMEOUT;
```

Comment:
waitTimeout is no longer needed. You can also remove WAIT_TIMEOUT and WAIT_DELTA.